rules.txt

```diff
@@ -506,6 +510,7 @@
 /news/oem/excessive-heat-warning-declared-for-philadelphia-july-6th-9th 301 /news/oem/excessive-heat-declared-for-philadelphia-summer-2016/
 /news/office-of-immigrant-affairs/immigration-sanctuary-cities-city-of-philadelphia-action-guide 301 /2018-01-08-immigration-policies/
 /news/office-of-sustainability/environment-sustainability-city-of-philadelphia-action-guide 301 /2017-11-29-environmental-protection-during-the-trump-administration/
+/next-?level                              301  /programs/adult-education/
```

Comment:
Do you need a second ? at the end for it to match on the word "level"? Same question for /substance-use
`next-?level?`

Reply:
Nope this should work. The ? just makes the character before it optional. I tested on staging